src/components/ContactUs/index.tsx

```diff
@@ -6,77 +6,61 @@ import { faEnvelope, faPhoneAlt } from '@fortawesome/free-solid-svg-icons';
 import { faFacebook, faInstagram } from '@fortawesome/free-brands-svg-icons';
 
 const ContactUsContainer = styled.div`
-display: flex;
-flex-flow: column nowrap;
-padding: 3rem .5rem;
-align-items: center;
-justify-content: center;
-width: 75%;
-margin: 0 auto;
+display:flex;
+flex-flow:row wrap;
```

Comment:
for mobile shouldnt be column?

Reply:
did not like having 4 elements as column